.gitmodules

```diff
@@ -4,3 +4,6 @@
 [submodule "lib/solmate"]
 	path = lib/solmate
 	url = https://github.com/transmissions11/solmate
+[submodule "lib/nst"]
```

Comment:
Wondering if worth to add the dependency and update solidity instead of just using deployed DAI for now and switch it NST whenever is deployed, which should be soon. Even if not active, as soon as we have the deployed contract we can run the script in the test to make it to work.

Reply:
Hmm ok maybe is too weird joining and exiting the same token, but a couple of mocks are an option as well as temporary option.

Reply:
In the interest of time, I would like to break the no dependency rule in this case.